content/doc/developer/publishing/releasing-cd.adoc

```diff
@@ -304,7 +304,13 @@ mvn -Dset.changelist \
 
 == Troubleshooting
 
-Check that `MAVEN_TOKEN` and `MAVEN_USERNAME` appear under Repository secrets and that your pull request has a label indicating it was of interest to users (e.g., enhancement rather than chore).
+Check that `MAVEN_TOKEN` and `MAVEN_USERNAME` appear under Repository secrets.
+
+=== No release has been triggered after merging a pull request
+
+Ensure that the pull request has a label indicating it was of interest to users (e.g., `enhancement` rather than `chore`).
+
+See the complete list of label categories in https://github.com/jenkinsci/.github/blob/master/.github/release-drafter.yml.
```

Comment:
might be worth mentioning what is interesting? as developer is interesting but may not be known as such?

Reply:
Added the link to the interesting categories in https://github.com/jenkins-infra/jenkins.io/pull/7686/commits/847dd874e812fbd542e3665ea5f67f68fdbc01d9